polaris.shopify.com/src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,59 @@
+import Link from 'next/link';
+import {useRouter} from 'next/router';
+import navJSON from '../../../.cache/nav.json';
+import {NavJSON} from '../../types';
+import get from 'lodash.get';
+
+import styles from './Breadcrumbs.module.scss';
+import {uppercaseFirst} from '../../utils/various';
+
+const nav = navJSON as NavJSON;
+
+interface Props {}
+
+function Breadcrumbs({}: Props) {
+  const {asPath} = useRouter();
+
+  let segments: {url: string; text: string}[] = [
```

Comment:
```suggestion
  const segments: {url: string; text: string}[] = [
```

I think this can be const because we are not reassigning a value to it
The `.push` in line 35 is ok, it doesn't create a new array